backend/src/apiserver/storage/object_store.go

```diff
@@ -126,6 +127,10 @@ func buildPath(folder, file string) string {
 	return folder + "/" + file
 }
 
-func NewMinioObjectStore(minioClient MinioClientInterface, bucketName string, baseFolder string, disableMultipart bool) *MinioObjectStore {
-	return &MinioObjectStore{minioClient: minioClient, bucketName: bucketName, baseFolder: baseFolder, disableMultipart: disableMultipart}
+func NewMinioObjectStore(minioClient MinioClientInterface, region string, bucketName string,
+	baseFolder string, disableMultipart bool) *MinioObjectStore {
+	return &MinioObjectStore{minioClient: minioClient, region: region,
```

Comment:
here it is just called region instead of serviceRegion. We should strive for a bit more consistency.